packages/ako-operator-v2/kbld-config.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: kbld.k14s.io/v1alpha1
 kind: Config
 overrides:
   - image: ako-operator:latest
-    newImage: "projects-stg.registry.vmware.com/tkg/sandbox/ako-operator:v1.7.0_vmware.1"
+    newImage: "projects-stg.registry.vmware.com/tkg/sandbox/ako-operator:v1.8.0_vmware.1"
```

Comment:
Unless we manually push one, I think we don't need to change the tag here.
The tag will only be used in upstream.
When building in bolt, it will overwrite the version

Reply:
Otherwise the test will fail
```
Error: couldn't generate imgpkg lock output file: couldn't generate imgpkg lock output file: couldn't run kbld command to generate imgpkg lock output file: kbld: Error: 
- Resolving image 'ako-operator:latest': GET https://projects-stg.registry.vmware.com/v2/tkg/sandbox/ako-operator/manifests/v1.8.0_vmware.1: NOT_FOUND: artifact tkg/sandbox/ako-operator:v1.8.0_vmware.1 not found
```